app/controllers/games_controller.rb

```diff
@@ -0,0 +1,49 @@
+class GamesController < ApplicationController
+
+  def index
+    @games = Game.where(user_id: current_user).order(created_at: :desc)
```

Comment:
zeigt nur Games, wo du Host bist

versuche es über
`@games = policy_scope(Game)`
das müssen wir dann noch in der policy definieren